src/libmongoc/tests/test-mongoc-client-side-encryption.c

```diff
@@ -135,93 +140,139 @@ _check_bypass (mongoc_auto_encryption_opts_t *opts)
    }
 }
 
-/* Convenience helper for creating KMS providers doc */
 static bson_t *
-_make_kms_providers (bool with_aws, bool with_local)
+_make_aws_kms_provider (bson_t *kms_providers)
 {
-   bson_t *kms_providers = bson_new ();
+   char *aws_secret_access_key;
+   char *aws_access_key_id;
 
-   if (with_aws) {
-      char *aws_secret_access_key;
-      char *aws_access_key_id;
-      char *azure_tenant_id;
-      char *azure_client_id;
-      char *azure_client_secret;
-      char *gcp_email;
-      char *gcp_privatekey;
-
-      aws_secret_access_key =
-         test_framework_getenv ("MONGOC_TEST_AWS_SECRET_ACCESS_KEY");
-      aws_access_key_id =
-         test_framework_getenv ("MONGOC_TEST_AWS_ACCESS_KEY_ID");
-
-      if (!aws_secret_access_key || !aws_access_key_id) {
-         fprintf (stderr,
-                  "Set MONGOC_TEST_AWS_SECRET_ACCESS_KEY and "
-                  "MONGOC_TEST_AWS_ACCESS_KEY_ID environment "
-                  "variables to run Client Side Encryption tests.");
-         abort ();
-      }
+   aws_secret_access_key =
+      test_framework_getenv ("MONGOC_TEST_AWS_SECRET_ACCESS_KEY");
+   aws_access_key_id = test_framework_getenv ("MONGOC_TEST_AWS_ACCESS_KEY_ID");
+   if (!aws_secret_access_key || !aws_access_key_id) {
+      fprintf (stderr,
+               "Set MONGOC_TEST_AWS_SECRET_ACCESS_KEY and "
+               "MONGOC_TEST_AWS_ACCESS_KEY_ID environment "
+               "variables to run Client Side Encryption tests.");
```

Comment:
```suggestion
               "variables to enable CSFLE tests..");
```

For consistency with error messages below.